app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,48 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def edit
+    load_partner
+    load_associations
+  end
+
+  def update
+    load_partner
+    load_associations
```

Comment:
Right, as per their example:
```ruby
# controller
def edit
  @aggregate = FacadeClass.new(get_partner)
end

private
def get_partner
  Partner.find(params[:id])
end

# facade_class.rb
class FacadeClass
  def initialize(partner)
    @partner = partner
  end

  def provinces
    Province.all
  end

  def statuses
    Status.all
  end
end

# _form.html.erb
@aggregate.partner.name # not a violation of Demeter!
@aggregate.provinces
@aggregate.statuses
```

I see value in a single "package" being passed from controller to the view